### PR TITLE
Fixed isHasKey and added trimWhitespaces

### LINKED ISSUE #479: implement isHasKey
There is currently a method called `isHasKey` that lives in `/lib/isHasKey.js`.
It is incomplete and needs to be fixed!

This method will return true if the input object or the dictionary contains the input key. In the other hand, return false if the input object does not contain the input key. e.g. isHasKey({"key":"value"}, "key") => true

### Acceptance Criteria:
1. Running `yarn test isHasKey` results in tests passing.
2. You have written a skeleton method for someone else to work on.
3. You have written tests surrounding your skeleton method.
4. Running `yarn lint` does not print any errors to the console!
5. Optional: write one or two more tests surrounding the method you are solving to account for potential edge cases.

### Please include the skeleton of a new method + an accompanying test for someone else to work on at the time of creating a pull request! A pull request will most likely be denied if it does not contain a skeleton method for someone else to work on! For more information, please read the [Contributing Guide](https://github.com/BlakeGuilloud/ganon/blob/master/CONTRIBUTING.md).

Thank you so much for your contribution!

### PR DESCRIPTION
**Pull request by polo123qwe**
Closes #479 

- [x] Running `yarn lint` does not trigger any linter errors
- [x] The test of the method you have fixed is passing
- [x] You have written a new skeleton method for someone else to work on!
- [x] You have written tests to accompany your skeleton method

### CODE REVIEW

File: test/trimWhitespaces.test.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+const { trimWhitespaces } = require("../lib");
```
Comment (ktilcu) [inline test/trimWhitespaces.test.js]: I think this would be better referred to as `removeWhitespace`. Trim would imply just removing whitespace from the ends.

Comment (polo123qwe) [reply test/trimWhitespaces.test.js]: Should be fixed now

**Comment by ktilcu** [comment]
Bingo! Thanks! Could you create a new issue too. It would be really helpful.
